### PR TITLE
fix: update Agoric explorer URL

### PR DESCRIPTION
**Pull request by nfl0**
This commit updates the explorer URLs for the Agoric network and its corresponding testnet (devnet) in the 'networks.ts' configuration file. The change was necessitated by the sunset announcement, as specified in the tweet from BigDipper (https://twitter.com/bigdipperlive/status/1766162059690119486).

### CODE REVIEW

File: modules/statics/src/networks.ts [hunk 1]
```diff
@@ -677,13 +677,13 @@ class HashTestnet extends Testnet implements AccountNetwork {
...
   family = CoinFamily.BLD;
-  explorerUrl = 'https://bigdipper.live/agoric/transactions/';
+  explorerUrl = 'https://agoric.explorers.guru/transaction/';
 }
 
 class BldTestnet extends Testnet implements AccountNetwork {
   name = 'Testnet Agoric';
   family = CoinFamily.BLD;
-  explorerUrl = 'https://emerynet.explorer.agoric.net/agoric/tx/';
+  explorerUrl = 'https://devnet.agoric.explorers.guru/transaction/';
```
Comment (mukeshsp) [inline modules/statics/src/networks.ts:686]: Why do you want to point on devnet ? we are using emerynet for testing.
```suggestion
  explorerUrl = 'https://devnet.agoric.explorers.guru/transaction/';
```

Comment (nfl0) [reply modules/statics/src/networks.ts:686]: Apologies for the confusion, I accidentally replaced the Emerynet chain explorer URL with the Devnet chain explorer URL in the configs file. I'll revert this change promptly.

**Comment by nfl0** [comment]
@mukeshsp

**Review by alebusse — APPROVED** [review]

**Comment by nfl0** [comment]
@mukeshsp

**Comment by nfl0** [comment]
@alebusse

**Comment by nfl0** [comment]
Could you please assist with CI test failures? Thanks

**Review by mukeshsp — APPROVED** [review]

**Review by ravneet-bitgo — DISMISSED** [review]

**Comment by alebusse** [comment]
@nfl0 rebase your branch with current master branch and push again, tests are failing probably because your branch is outdated.

**Comment by nfl0** [comment]
@alebusse done. please approve

**Comment by alebusse** [comment]
@nfl0 could you please squash the commits and leave only one?

**Review by ravneet-bitgo — COMMENTED** [review]
flush

**Comment by nfl0** [comment]
roger

**Comment by mohammadalfaiyazbitgo** [comment]
created a duplicate pr https://github.com/BitGo/BitGoJS/pull/4747 since I'm unable to fetch the current one to squash and refresh this pr.

**Pull request closed without merging**